src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -13019,6 +13019,105 @@ private static Vector512<T> IsNegative<T>(Vector512<T> vector)
             return Vector512.LessThan(vector, Vector512<T>.Zero);
         }
 
+        /// <summary>poly = C0 + C1*r^3 + C2*r^5 + C3*r^7 + C4*r^9</summary>
+        private static Vector128<T> PolyEvalOdd9<T>(Vector128<T> r, Vector128<T> c0, Vector128<T> c1, Vector128<T> c2, Vector128<T> c3, Vector128<T> c4)
+        {
+            Vector128<T> r2 = r * r;
+            Vector128<T> r4 = r2 * r2;
+
+            Vector128<T> a0 = c2 * r2 + c1;
+            Vector128<T> a1 = a0 * r2 + c0;
+            Vector128<T> a2 = (c3 * r2 + c4 * r4) * r4;
+
+            return r * (a1 + a2);
+        }
+
+        /// <summary>poly = C0 + C1*r^3 + C2*r^5 + C3*r^7 + C4*r^9</summary>
```

Comment:
Nit: I'm guessing this notation is borrowed from elsewhere, but convention for polynomials is that coefficient labels correspond to their degree, i.e.

```C#
poly = C0 + C3*r^3 + C5*r^5 + C7*r^7 + C9*r^9
```

It would probably also make the implementations easier to read.